src/server/scripts/Commands/cs_character.cpp

```diff
@@ -38,7 +38,7 @@ class character_commandscript : public CommandScript
             { "delete",        SEC_CONSOLE,          true,  &HandleCharacterDeletedDeleteCommand,  "" },
             { "list",          SEC_ADMINISTRATOR,    true,  &HandleCharacterDeletedListCommand,    "" },
             { "restore",       SEC_ADMINISTRATOR,    true,  &HandleCharacterDeletedRestoreCommand, "" },
-            { "old",           SEC_CONSOLE,          true,  &HandleCharacterDeletedOldCommand,     "" },
+            { "purge",           SEC_CONSOLE,          true,  &HandleCharacterDeletedOldCommand,     "" },
```

Comment:
you could respect the indentation :P

Reply:
Yep sorry totaly missed that!

Reply:
HandleCharacterDeletedOldCommand -> HandleCharacterDeletedPurgeCommand